src/client/components/componentWithSliders/savePressetModalBlock/savePressetModalBlock.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import ModalSavePressetesWindow from './modalSavePressetesWindow';
+import BackgroundUnderModelWindow from './backgroundUnderModelWindow';
+
+class SavePressetModalBlock extends Component {
+    autoFocusModalSavePressetesWindow = (elementFromRef) => {
+      if (elementFromRef) {
+        elementFromRef.focus();
+      }
+    }
+
+    gatherPressetesValuesAndSendToBD = () => {
```

Comment:
Are you need this now, or it can be added later, when logic is needed ?

Reply:
deleted